UI/src/view/LinearAssetLayer.js

```diff
@@ -219,6 +221,17 @@ window.LinearAssetLayer = function(params) {
     }
   }
 
+  function onDrawEnd(linearAssets) {
```

Comment:
This function should be edited. onDragEnd function on this file has the same code